src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -210,7 +210,10 @@ override async void OnActivated(IActivatedEventArgs e)
 					$"PreviousState - {e.PreviousExecutionState}, " +
 					$"Uri - {protocolActivatedEventArgs.Uri}",
 					"Application activated via protocol");
+#if !__SKIA__
```

Comment:
It's probably preferable to use `ApiInformation` instead, so it lights up automatically. Same for the other location.